Terrace-Svelte/src/components/Home/Weather.svelte

```diff
@@ -1,100 +1,28 @@
-        return Math.round(1.8*(k-273) + 32);
-    }
-
-
-        // For sunrise/sunset
-    function epochToReadable(epoch) {
-        let date = new Date(epoch * 1000);
-        let minutes
-        // Handles for missing 0 in times like 5:06
-        if (date.getMinutes() < 10) {
-            minutes = "0" + date.getMinutes()
-        } else {
-            minutes = date.getMinutes()
-        }
-        // Converts from 24 hour time to 12 hour time
-        if (date.getHours() > 12) {
-            return (date.getHours() - 12) + ":" + minutes;
-        } else {
-            return date.getHours()+ ":" + minutes;
-        }
-    }
-
-    // For helping getting local time...
-    const addZeroIfNeeded = (num) => {
-        return (num < 10) ? '0' + num : num.toString();
-    }
-
-    let daysOfWeek = {
-        1: "Monday",
-        2: "Tuesday",
-        3: "Wednesday",
-        4: "Thursday",
-        5: "Friday",
-        6: "Saturday",
-        0: "Sunday"
-    }
-
-    let months = {
-        1: "January",
-        2: "February",
-        3: "March",
-        4: "April",
-        5: "May",
-        6: "June",
-        7: "July",
-        8: "August",
-        9: "September",
-        10: "October",
-        11: "November",
-        12: "December"
-    }
-
-    let focusedNav
-    const navStore = currentNav.subscribe(value =>{
-        focusedNav = value;
-    });
-
-    // Interval Timer to request Stats
-    setInterval(updateTime, 1000);
-
-    function updateTime() {
-        if (focusedNav ==="Home") {
-            let myDate = new Date();
-            let hours = addZeroIfNeeded(myDate.getHours());
-            if (hours == 0) {
-                hours = 12
-            }
-            if (hours > 12) {
-                hours = hours - 12
-            }
-            let mins = addZeroIfNeeded(myDate.getMinutes());
-            let seconds = addZeroIfNeeded(myDate.getSeconds());
-            let day = myDate.getDay();
-            let dateDay = myDate.getUTCDate();
-            let month = myDate.getMonth();
-            document.getElementById("time").innerHTML = `${hours}:${mins}:${seconds}`;
-            document.getElementById("date").innerHTML = `${daysOfWeek[day]}, ${months[month + 1]} ${dateDay} ${myDate.getFullYear()}`;
-        }
-    }
-
+    // import { currentNav } from "../../stores";
+    let lat = "39.983334",
+        long = "-82.983330",
+        weatherApiKey = "dbd30986d45f5c219692ea5d83e34a51",
+        weatherDescription, temp, feelsLike, humidity, sunrise, sunset, wind, timeZone, iconUrl, icon, formattedTime = "", formattedDate = "", focusedNav,
+        elmWeatherIcon, elmWeatherIconSrc, 
+        elmTemperature = "73°F", 
+        elmFeelsLike = "60", 
+        elmHumidity = "50%", 
+        elmWindSpeed = "12 mph", 
+        elmSunrise = "7:00", 
+        elmSunset = "9:00";
+
+    // const navStore = currentNav.subscribe(value =>{
+    //     focusedNav = value;
+    // });
+
+    setInterval(() => {
```

Comment:
the focusedNav check here needs to be re-implemented but the code for this isn't pushed.
the rest of this code was condensed to date strings.